Automattic/jetpack
# Add zendesk chat authentication

## Proposed changes:

 * Update zendesk chat component to accept an authentication token


### Other information:


## Jetpack product discussion
P2: p7pQDF-8rT-p2#comment-23449

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

- Create a JN or an Atomic Ephemeral site with Jetpack Beta installed
- Set the Jetpack Beta branch to `add/zendesk-chat-authentication`
- Connect Jetpack to your WordPress.com account
- Visit `/wp-admin/admin.php?page=my-jetpack`
- If the chat is available (the chat bubble is present), click to start a chat
- Confirm it doesn't request your Name/Email address
- If you start a new chat, or if you already have a chat in the history, confirm that you can see the same messages in the chat on Jetpack.com or cloud.jetpack.com/pricing
- If you have a ZD account, you can search for the ticket related to the chat above in ZD to confirm you're authenticated



<img width="399" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/a5be97a8-1562-4ccb-a4ee-8d1309b3827a">
